deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo "project build with maven 3.8.5"
```

Comment:
to ważne? jeśli tak to polecam spojrzeć na mvnw

Reply:
Dodając wrappera(./mvnw/wrapper i mvnw) każdy mógłby uruchomić aplikacje bez zainstalowanego maven'a, nie zawracając sobie głowy wersją.